packages/store-indexer/src/sqlite/apiRoutes.ts

```diff
@@ -44,5 +45,50 @@ export function apiRoutes(database: BaseSQLiteDatabase<"sync", any>): Middleware
     }
   });
 
+  router.post("/q", async (ctx) => {
+    if (!enableQueryEndpoint) {
+      ctx.status = 404;
+      ctx.body = JSON.stringify({ error: "Query endpoint is not enabled" });
+      return;
+    }
+
+    try {
+      const queries = Array.isArray(ctx.request.body) ? ctx.request.body : [];
```

Comment:
does dozer allow for more than one query at a time? are they executed in the same db tx?